catalog/app/components/Preview/types.js

```diff
@@ -1,28 +1,5 @@
 import tagged from 'utils/tagged'
 
-/*
-ParquetMeta: {
```

Comment:
these should be preserved probably (it's not related to this PR, but anyways, it looks like a regression introduced by the previous one)

Reply:
Started to work on resolving regression